### PR TITLE
added phase to FrequencyRampPulse

### PR DESCRIPTION
**Pull request by RostyslavSavytskyy**

### CODE REVIEW

File: silq/pulses/pulse_types.py [hunk 1]
```diff
@@ -603,7 +602,7 @@ def __init__(self,
                                          set_cmd=None, vals=vals.Enum('relative',
                                                                       'absolute'))
         self._connect_parameters_to_config(
-            ['frequency', 'phase', 'power', 'amplitude', 'phase', 'offset',
```
Comment (nulinspiratie) [inline silq/pulses/pulse_types.py:606]: Why is phase removed?

File: silq/pulses/pulse_types.py [hunk 2]
```diff
@@ -684,6 +683,11 @@ class FrequencyRampPulse(Pulse):
             initial frequency or stay at current frequency. Useful if the
             pulse doesn't immediately stop at the end (this depends on how
             the corresponding instrument/interface is programmed).
+        phase: Pulse phase. By default is set to zero.
+        phase_reference: What point in the the phase is with respect to.
+            Can be two modes:
+                - 'absolute': phase is with respect to `Pulse.t_start`.
```
Comment (nulinspiratie) [inline silq/pulses/pulse_types.py:689]: Absolute and relative are swapped here

File: silq/pulses/pulse_types.py (hunk 1 above)
Comment (RostyslavSavytskyy) [reply silq/pulses/pulse_types.py:606]: because it's repeated twice there

File: silq/pulses/pulse_types.py (hunk 2 above)
Comment (RostyslavSavytskyy) [reply silq/pulses/pulse_types.py:689]: I copied this from SinePulse - do you have different definitions for different pulses?

Comment (RostyslavSavytskyy) [reply silq/pulses/pulse_types.py]: Swapped them and also for SinPulse as well

**Pull request merged**